gh-gl-sync/SpackCIBridge.py

```diff
@@ -400,10 +467,19 @@ def sync(self):
             # Setup the local repo with two remotes.
             self.setup_git_repo()
 
+            # Find the currently running develop pipeline, if any, and get the sha
+            currently_running_develop_sha = None
+            develop_pipelines = self.get_pipelines_for_branch("develop", self.time_threshold_dev)
```

Comment:
Can we make this `self.main_branch_name` or so rather than hardcoding develop? I like the idea of keeping this sync script generally useful (not just for Spack).

Reply:
Yes, makes sense.  How about another command line option?  Like maybe `--main-branch` which defaults to `develop`?

Reply:
I like this idea